config/src/main/java/org/springframework/security/config/annotation/web/configurers/CsrfConfigurer.java

```diff
@@ -76,6 +76,8 @@
     private CsrfTokenRepository csrfTokenRepository = new HttpSessionCsrfTokenRepository();
     private RequestMatcher requireCsrfProtectionMatcher = CsrfFilter.DEFAULT_CSRF_MATCHER;
     private List<RequestMatcher> ignoredCsrfProtectionMatchers = new ArrayList<RequestMatcher>();
+	private String cookieName;
+	private String cookiePath;
```

Comment:
Tabs instead of spaces. If @rwinch has a preference, he probably has a formatter in his IDE. Mine obviously doesn't match his.